lib/node.js

```diff
@@ -935,6 +935,7 @@ declare class http$ServerResponse extends stream$Writable {
 declare module "http" {
   declare class Server extends net$Server {
     listen(port: number, hostname?: string, backlog?: number, callback?: Function): Server;
+    listen(port: number, callback?: Function): Server;
```

Comment:
Looks like this is redundant? If you pass just a `port: number` and a `callback?: Function`, the line above should be able to match it.